inlong-manager/manager-plugins/base/src/main/assembly/assembly.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements. See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License. You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<assembly
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://maven.apache.org/ASSEMBLY/2.1.0"
+        xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.1.0 http://maven.apache.org/xsd/assembly-2.1.0.xsd">
+    <id>plugins</id>
+    <formats>
+        <format>dir</format>
+    </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <fileSets>
+        <fileSet>
+            <directory>${build.directory}</directory>
+            <outputDirectory>./</outputDirectory>
+            <includes>
+                <include>${project.artifactId}-${project.version}.jar</include>
+            </includes>
+        </fileSet>
+
+        <!-- Plugins Flink v1.13 -->
+        <fileSet>
+            <directory>../manager-plugins-flink-v1.13/target</directory>
+            <outputDirectory>./</outputDirectory>
+            <includes>
+                <include>manager-plugins-flink-v1.13.jar</include>
+            </includes>
+        </fileSet>
+
+        <!-- Plugins Flink v1.15 -->
+<!--        <fileSet>-->
```

Comment:
Is it possible to add an assembly-v1.15.xml file?

Reply:
![image](https://github.com/apache/inlong/assets/58519431/31963929-8979-4a65-a653-289371dccd80)

Both Flink 1.13 and 1.15 versions will be packaged at one time and placed in different directories. 

When starting the manager, set the classpath according to the Flink version, so we might not need to manage Flink 1.15 separately.